stdlib/3.4/asyncio/events.pyi

```diff
@@ -123,7 +125,9 @@ class AbstractEventLoop(metaclass=ABCMeta):
     @coroutine
     def create_datagram_endpoint(self, protocol_factory: _ProtocolFactory,
                                  local_addr: str = ..., remote_addr: str = ..., *,
-                                 family: int = ..., proto: int = ..., flags: int = ...) -> Generator[Any, None, _TransProtPair]: ...
+                                 family: int = ..., proto: int = ..., flags: int = ...,
+                                 sock: socket = ..., reuse_address: Optional[bool] = ...,
```

Comment:
You're missing the `allow_broadcast` argument (and sock is out of order, though that doesn't matter for keyword-only arguments).